### DISCUSSION TITLE
Week 4 Games and Project

### DISCUSSION THREAD

**Discussion by gabebott** [USU-ITLS-x265-23-summer/class-repo#102] [General]
I thought that the CSS Diner game was very helpful and made me feel a lot more comfortable coding. I think I am hopefully starting to get the hang of it. I really liked how the game built upon previous knowledge and had a level where it didn't give any examples or instruction. I thought this was a good challenge and a test to see if I was retaining the previous information. One thing that I think they could do better on is giving more help. There were a few different levels that I struggled on particularly 16 and 19. I feel like these levels could have used some more instruction, or some other resources to access if you got stuck. I ended up using github to get help with these issues, but I think CSS Diner itself should have some more resources for when you get stuck.

I also played the Elevator Saga game. I think this game really excelled in helping you understand the controls and the objectives of the game. I don't think I found this one as informative as CSS Diner, but I thought the premise of the game was fun and it helped me get more practice writing in this new language which is never a bad thing!

As far as my project goes, I have thought about how I am going to limit down my list of potential projects. I think I got a little overambitious with some of my ideas, so I tried to pick projects that would challenge me, but still be good for a beginner. The four that I have decided will move on to the next round of deliberation are:

1. Personal Website Portfolio: a simple personal website using HTML, CSS, and JavaScript. Showcase UX portfolio projects on different web pages.
2. To-Do List App: Develop a basic to-do list application using HTML, CSS, and JavaScript. Allow users to add, remove, and mark tasks as complete. You can enhance it with features like local storage to save tasks.
3. Memory Game: Build a memory game using HTML, CSS, and JavaScript. Create a grid of cards that the user can flip over to match pairs. Add features like a timer, score counter, and different levels of difficulty.
4. Movie Quote Generator: Create a simple quote generator using HTML, CSS, and JavaScript. Fetch quotes from movies and tv and display them on the webpage. Add a button to generate a new quote dynamically.

**Comment by metasean** [comment]
>I think I got a little overambitious with some of my ideas

I'm glad that the combination of project ideation, while working through the lessons, is helping you refine your expectations.
